code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -605,7 +605,11 @@ GLOBAL_VAR(station_nuke_source)
 
 /proc/KillEveryoneOnStation()
 	for(var/mob/living/victim as anything in GLOB.mob_living_list)
-		if(victim.stat != DEAD && is_station_level(victim.z))
+		var/turf/M = get_turf(victim)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names

bad var names

Reply:
Sorry my bad, I will get on it ASAP.